<!doctype html>
<html>
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
<link rel="stylesheet" href="css/theme.css" media="screen" type="text/css" />
<title>Welcome to ClassifyMM</title>

</head>
<body>
<h1>Welcome to ClassifyMM</h1>

<hr />

<h2>Abstruct</h2>

<p>Multiple myeloma is a cancer of plasma cells that normally produces antibodies. Aneuploidy, the alterations in the number of whole or partial chromosomes, is consistently observed in many cancer types including multiple myeloma. According to the status of chromosomes in multiple myeloma, this cancer can be subdivided into two main subtypes: hyperdiploid multiple myeloma (HMM) and non-hyperdiploid multiple myeloma (NHMM). The two subtypes have different survival prognosis, possibly due to their different but converging paths to myeloma. The existing method to identify the two subtypes is fluorescence in situ hybridization (FISH), and there is no effective method using gene expression profiles to classify the two subtypes. We have built a nearest-neighbor based classification method to separate multiple myeloma into HMM and NHMM with gene expression profiles, providing a useful tool for further studying of multiple myeloma. We also provide publicly the R packages and the processed training data sets <a href="http://hmmc.github.com">here</a>.</p>

<p><code>Classifying hyperdiploidy status of multiple myeloma samples using gene expression profiles</code></p>

<h2>Download</h2>

<p>Packages: Windows <a href="data/ClassifyMM_1.0.zip">ClassifyMM_1.0.zip</a> Mac <a href="data/ClassifyMM_1.0.tar.gz">ClassifyMM_1.0.tar.gz</a></p>

<p>Training data: <a href="data/TrainData.tar.bz2">TrainData.tar.bz2</a></p>

<p>Test data(GSE6401, not whole): <a href="data/GSE6401_RAW.tar.bz2">GSE6401_RAW.tar.bz2</a></p>

<p>Source code: <a href="data/mmsc.r">mmsc.r</a></p>

<h2>Usage</h2>

<h3><em>install</em></h3>

<p>Package affy</p>

<pre><code>&gt; source("http://bioconductor.org/biocLite.R")
&gt; biocLite("affy")
</code></pre>

<p>Package AnnotationDbi</p>

<pre><code>&gt; source("http://bioconductor.org/biocLite.R")
&gt; biocLite("AnnotationDbi")
</code></pre>

<p>Data CDF file <a href="http://brainarray.mbni.med.umich.edu/Brainarray/Database/CustomCDF/CDF_download.asp">download</a></p>

<p>Exmaple for download "HG-U133_Plus2" array:First, verify you OS and R version. Here my work on Mac OS and R2.15. Second, click the link above, find column names R.15,BioCxxx and raw names REFSEQ, click "click me". Last, look for ran names "HG-U133_Plus2" and column names "source"(win32 for windows), click "C".</p>

<pre><code>run R GUI --&gt; package/data --&gt; package installer --&gt; local source package.
</code></pre>

<p><code>Note:Do not use command install.packages(), It does not work!</code></p>

<p>Install ClassifyMM</p>

<pre><code>run R GUI --&gt; package/data --&gt; package installer --&gt; local source package.
</code></pre>

<h3>Input for main function mmsc()</h3>

<p><em>Raw_Dir</em> | <em>Exp_File</em> : Raw_Dir for raw data directory and Exp_File for expression data file name. The format of raw data is compressed .cel from GEO database by affy matrix technology. You can input raw data directly, then the program will first calculate the expression by R package <em>affy</em>. Otherwise, you must provide common style expression data. You can't provide both Raw_Dir and Exp_File.</p>

<p><em>CDF_Name</em> : Refer to CDF names used by package "affy".</p>

<p>[<em>Tra_Dir</em>] : A directory contain the training data. By default it's "./TrainData".</p>

<p>[<em>Out_Dir</em>] : A directory contain all of result and temp file. By default it's "./OutPut".</p>

<p>[<em>K_Const</em>] : The number of neighbors used in KNN classifer. By default it's 16.</p>

<p>[<em>Tun_Time</em>] : By default it's 1:140.</p>

<p><code>Note:[ parameter ] parameter is optional.</code></p>

<h3>Output</h3>

<p>ExprData_MM_sort.txt :</p>

<p>Expression_Temp.XLS :</p>

<p>MeanDistPlot :</p>

<p>MMSCresult_k16.txt :</p>

<h3>example</h3>

<pre><code>#set workshop
setwd("~/Documents/YingxiangL/");

#overview the file and directory
dir();

#load package
library(ClassifyMM);

#analysie data
mmsc(Raw_Dir=“GSE6401_RAW/”, Tra_Dir="TrainData/", CDF_Name="HGU133a_hs_refseq");
</code></pre>

<h3>screenshot</h3>

<p><img src="image/cdf1.png" alt="image" />
<img src="image/cdf2.png" alt="image" />
<img src="image/dir.png" alt="image" />
<img src="image/run.png" alt="image" />
<img src="image/result.png" alt="image" /></p>

<h2>About</h2>

<p>Developed by <a href="http://www.chenglilab.org">Cheng Li Lab</a>.</p>
</body>
</html>
